### PR TITLE
Add sample for the Scalable Video Coding (SVC) Extension for WebRTC

### PR DESCRIPTION
**Pull request by murillo128**
**Description**
Add sample for testing the Scalable Video Coding (SVC) Extension for WebRTC 
https://www.w3.org/TR/webrtc-svc/

**Purpose**
Check and test the different scalability modes supported by each codec.

### CODE REVIEW

**Comment by murillo128** [comment]
@alvestrand could you review it, please?

File: src/content/extensions/svc/index.html [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+        <div>Bitrate</div>
+        <canvas id="bitrateCanvas"></canvas>
+    </div>
+    <div class="graph-container" id="packetGraph">
+        <div>Packets sent per second</div>
+        <canvas id="packetCanvas"></canvas>
+    </div>
+
+
+    <a href="https://github.com/webrtc/samples/tree/gh-pages/src/content/peerconnection/change-codecs"
```
Comment (alvestrand) [inline src/content/extensions/svc/index.html]: Fix source link to point to the right extension

File: src/content/extensions/svc/js/main.js [hunk 2]
```diff
@@ -0,0 +1,357 @@
...
+      const option = document.createElement('option');
+      option.value = (codec.mimeType + ' ' + (codec.sdpFmtpLine || '')).trim();
+      option.innerText = option.value;
+      codecPreferences.appendChild(option);
+    });
+    codecPreferences.addEventListener('change', (event) => {
+      const [mimeType, sdpFmtpLine] = event.target.value.split(' ');
+      const selectedCodecIndex = codecs.findIndex(c => c.mimeType === mimeType && c.sdpFmtpLine === sdpFmtpLine);
+      const selectedCodec = codecs[selectedCodecIndex];
+      scalabilityMode.innerHTML = "";
```
Comment (alvestrand) [inline src/content/extensions/svc/js/main.js]: this is an antique way of clearing the members of a node. can't you use scalabilityMode.children = null or something like that?

File: src/content/extensions/svc/index.html [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+<body>
+
+<div id="container">
+    <h1><a href="//webrtc.github.io/samples/" title="WebRTC samples homepage">WebRTC samples</a>
+        <span>Peer connection</span></h1>
+
+    <p>This sample shows how to setup a connection between two peers using
+        <a href="https://developer.mozilla.org/en-US/docs/Web/API/RTCPeerConnection">RTCPeerConnection</a> and
+        choose the preferred video codec to use and scalability mode when the <a href="https://www.w3.org/TR/webrtc-svc/">Scalable Video Coding (SVC) Extension</a> is available.
+</p>
```
Comment (alvestrand) [inline src/content/extensions/svc/index.html:41]: Should you add a line saying "Codec and scalability mode must be chosen before the call"?

I see that you don' t have any code for changing the scalability mode while the call is active.

**Review by alvestrand — COMMENTED** [review]
Just a few nits. Apologies for not having reviewed earlier!

File: src/content/extensions/svc/index.html (hunk 1 above)
Comment (murillo128) [reply src/content/extensions/svc/index.html]: fixed

File: src/content/extensions/svc/js/main.js (hunk 2 above)
Comment (murillo128) [reply src/content/extensions/svc/js/main.js]: changed

File: src/content/extensions/svc/index.html (hunk 3 above)
Comment (murillo128) [reply src/content/extensions/svc/index.html:41]: I have disabled the scalabilityMode select while the call is ongoing in the same way as the codecPreferences select is.

File: src/content/extensions/svc/css/main.css [hunk 4]
```diff
@@ -0,0 +1,55 @@
+/*
+ *  Copyright (c) 2015 The WebRTC project authors. All Rights Reserved.
```
Comment (KaptenJansson) [inline src/content/extensions/svc/css/main.css]: Nit: the copyright year should be set to the current year for new files.

File: src/content/extensions/svc/index.html [hunk 5]
```diff
@@ -0,0 +1,85 @@
+<!DOCTYPE html>
+<!--
+ *  Copyright (c) 2015 The WebRTC project authors. All Rights Reserved.
```
Comment (KaptenJansson) [inline src/content/extensions/svc/index.html]: Nit: the copyright year should be set to the current year for new files.

File: src/content/extensions/svc/js/main.js [hunk 6]
```diff
@@ -0,0 +1,361 @@
+/*
+ *  Copyright (c) 2020 The WebRTC project authors. All Rights Reserved.
```
Comment (KaptenJansson) [inline src/content/extensions/svc/js/main.js]: Nit: the copyright year should be set to the current year for new files.

**Review by KaptenJansson — APPROVED** [review]
LGTM with nits

File: src/content/extensions/svc/index.html (hunk 5 above)
Comment (alvestrand) [inline src/content/extensions/svc/index.html]:
```suggestion
 *  Copyright (c) 2022 The WebRTC project authors. All Rights Reserved.
```

File: src/content/extensions/svc/css/main.css (hunk 4 above)
Comment (alvestrand) [inline src/content/extensions/svc/css/main.css]:
```suggestion
 *  Copyright (c) 2022 The WebRTC project authors. All Rights Reserved.
```

File: src/content/extensions/svc/js/main.js (hunk 6 above)
Comment (alvestrand) [reply src/content/extensions/svc/js/main.js]:
```suggestion
 *  Copyright (c) 2022 The WebRTC project authors. All Rights Reserved.
```

**Review by alvestrand — APPROVED** [review]

**Comment by alvestrand** [comment]
@murillo128 the linter has suddenly woken up on this CL. Can you fix?

**Comment by alvestrand** [comment]
@murillo128 ping on the lint issue....

**Comment by murillo128** [comment]
should be good now

**Pull request merged**